Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM node:12.10-slim
+FROM node:lts-slim
```

Comment:
What does this mean in terms of version? 
Will this automatically update to the latest LTS version or is LTS-slim a specific LTM version?

Reply:
It should automatically update.

Reply:
In that, case could you rig up this Dockerfile to a cron build https://docs.travis-ci.com/user/cron-jobs/ , run maybe weekly so we have some heads-up/record when/if this breaks.